parsers/jscript.c

```diff
@@ -267,14 +267,14 @@ static roleDefinition JsClassRoles [] = {
 };
 
 static kindDefinition JsKinds [] = {
-	{ true,  'f', "function",	  "functions",
-	  .referenceOnly = false, ATTACH_ROLES(JsFunctionRoles) },
+	{ true,  'f', "function",	  "functions"        },
 	{ true,  'c', "class",		  "classes",
 	  .referenceOnly = false, ATTACH_ROLES(JsClassRoles)    },
 	{ true,  'm', "method",		  "methods"          },
 	{ true,  'p', "property",	  "properties"       },
 	{ true,  'C', "constant",	  "constants"        },
-	{ true,  'v', "variable",	  "global variables" },
+	{ true,  'v', "variable",	  "global variables",
+	  .referenceOnly = false, ATTACH_ROLES(JsVariableRoles) },
```

Comment:
This change requires updating the stdout-expected.txt files under Tmain.
Run `make tmain` locally.

Reply:
Done